src/components/Modal.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { StyledModal } from "./Modal.styled";
+
+const Modal = ({
+  isShowing,
+  hide,
```

Comment:
modal is builded to always have `Cancel` button which is about to close modal so we don't need to pass another function  which does same action like `onCancel`.